checkstyle-tester/diff.groovy

```diff
@@ -340,23 +347,24 @@ def getCommitSha(commitId, repoType, srcDestinationDir) {
     return sha.replace('\n', '')
 }
 
-def getCloneCmd(repoType, repoUrl, srcDestinationDir) {
-    def cloneCmd = ''
-    switch (repoType) {
-        case 'git':
-            cloneCmd = "git clone $repoUrl $srcDestinationDir"
-            break
-        default:
-            throw new IllegalArgumentException("Error! Unknown $repoType repository.")
+def shallowCloneRepository(repoName, repoType, repoUrl, commitId, srcDir) {
+    def srcDestinationDir = getOsSpecificPath("$srcDir", "$repoName")
+    if (!Files.exists(Paths.get(srcDestinationDir))) {
+        def cloneCmd = getCloneShallowCmd(repoType, repoUrl, srcDestinationDir, commitId)
+        println "Cloning $repoType repository '$repoName' to $srcDestinationDir folder ..."
```

Comment:
```suggestion
        println "Shallow clone $repoType repository '$repoName' to $srcDestinationDir folder ..."
```

Reply:
I wish i had such eye for the detail. Thanks for pointing it out. Done.